CHANGELOG.md

```diff
@@ -55,6 +55,15 @@
 
 ## 6.3.2
```

Comment:
we want to move breaking into the next version so `7.0.0` here right? or are these pre-7 breaking changes?

Reply:
Yeah, I don't know why I keep messing up these changelog notes across the pull requests 😖 I'm guessing a bad rebase conflict resolution.